avro-converter/src/main/java/io/confluent/connect/avro/AvroData.java

```diff
@@ -161,6 +162,13 @@
         .getElementType();
   }
 
+  private static final HashMap<String, String> AVRO_LOGICAL_TYPE_NAMES_TO_CONNECT_LOGICAL_TYPE_NAMES = new HashMap<>();
+  static {
+    AVRO_LOGICAL_TYPE_NAMES_TO_CONNECT_LOGICAL_TYPE_NAMES.put("decimal", Decimal.LOGICAL_NAME);
```

Comment:
Shouldn't we need the reverse mapping as well? Otherwise when converting Connect -> Avro we don't capture these mappings, right? We need to set the `logicalType` field in the schema?